src/test/java/org/apache/commons/lang3/StringUtilsEqualsIndexOfTest.java

```diff
@@ -48,6 +49,8 @@
 
     private static final String[] FOOBAR_SUB_ARRAY = new String[] {"ob", "ba"};
 
+    private static final int NOT_FOUND = -1;
```

Comment:
This is already defined in `ArrayUtils`.

Reply:
true. changed